README.md

```diff
@@ -1,102 +1,63 @@
-`bison --version`
-(The result should be > 2.4.1 )
-
-** OS X has LDAP installed, but when compiling against it (/System/Library/Frameworks/LDAP.framework/Headers/ldap.h) you will get a `#include nested too deeply`, which is why you should install openldap.
-   
-* Create a build directory - either as a child of hpcc or elsewhere
-* cd to the build directory
-* Use clang to build the clienttools (gcc4.2 cores when compiling some of the sources):
-
-```
-   export CC=/usr/bin/clang 
-   export CXX=/usr/bin/clang++ 
-   cmake ../ -DICU_LIBRARIES=/usr/local/opt/icu4c/lib/libicuuc.dylib -DICU_INCLUDE_DIR=/usr/local/opt/icu4c/include -DLIBARCHIVE_INCLUDE_DIR=/usr/local/opt/libarchive/include -DLIBARCHIVE_LIBRARIES=/usr/local/opt/libarchive/lib/libarchive.dylib -DBOOST_REGEX_LIBRARIES=/usr/local/opt/boost/lib -DBOOST_REGEX_INCLUDE_DIR=/usr/local/opt/boost/include -DCLIENTTOOLS_ONLY=true -DUSE_OPENLDAP=true -DOPENLDAP_INCLUDE_DIR=/usr/local/opt/openldap/include -DOPENLDAP_LIBRARIES=/usr/local/opt/openldap/lib/libldap_r.dylib
-```
-
-* To build the makefiles just created above, run
-   make
-* Executables will be created in ./&lt;releasemode&gt;/bin and ./&lt;releasemode&gt;/libs
-* To create a .dmg to install, run
-   make package
- 
- 
+# Description / Rationale
+
+HPCC Systems offers an enterprise ready, open source supercomputing platform to solve big data problems. As compared to Hadoop, the platform offers analysis of big data using less code and less nodes for greater efficiencies and offers a single programming language, a single platform and a single architecture for efficient processing. HPCC Systems is a technology division of LexisNexis Risk Solutions.
+
+# Getting Started
+
+* [Learn about HPCC](http://hpccsystems.com/download/getting-started)
+* [Download](http://hpccsystems.com/download/free-community-edition)
+* [Installation and Running](http://hpccsystems.com/download/docs/installing-running-hpcc-platform)
+* [Build from Source](https://github.com/hpcc-systems/HPCC-Platform/wiki/Building-HPCC)
+
+# Architecture
+
+The HPCC Systems architecture incorporates the Thor and Roxie clusters as well as common middleware components, an external communications layer, client interfaces which provide both end-user services and system management tools, and auxiliary components to support monitoring and to facilitate loading and storing of filesystem data from external sources. An HPCC environment can include only Thor clusters, or both Thor and Roxie clusters. Each of these cluster types is described in more detail in the following sections below the architecture diagram.
+
+## [Thor](http://hpccsystems.com/FAQ/what-thor)
+
+Thor (the Data Refinery Cluster) is responsible for consuming vast amounts of data, transforming, linking and indexing that data. It functions as a distributed file system with parallel processing power spread across the nodes. A cluster can scale from a single node to thousands of nodes.
+
+* Single-threaded
```

Comment:
I think the distinction Brian is trying to make is that thor runs one query at a time while Roxie runs multiple. Internally, both use multiple threads (and multiple processes) to get the job done.